stories/atoms/cc-input-text.stories.js

```diff
@@ -154,7 +172,9 @@ export const allFormControls = allFormControlsStory;
 
 enhanceStoriesNames({
   defaultStory,
-  label,
+  required,
+  helpMessage,
+  errorMessage,
```

Comment:
Not sure about how to order stories :roll_eyes: 
error and help messages should be together but should they appear as one of the first stories like here or as one of the last instead ?

Reply:
It's fine like this :+1